### PR TITLE
add show all feature

### PR DESCRIPTION
**Pull request by samson84**

### CODE REVIEW

File: showAll.js [hunk 1]
```diff
@@ -6,6 +6,10 @@
 const getBeers = require('./data')
 
 function main() {
+  const beers = getBeers()
+  for(const beer of beers) {
```
Comment (samson84) [inline showAll.js:10]: lkjdsjlkdsjflkdsfj
:-1: :+1: @nemetabe

**Review by samson84 — COMMENTED** [review]
dsjkfjdslkfjsflksjfdlklkflkdsjfs

File: showAll.js [hunk 2]
```diff
@@ -6,6 +6,10 @@
 const getBeers = require('./data')
 
 function main() {
+  const beers = getBeers()
```
Comment (nemetabe) [inline showAll.js]: ;

**Review by nemetabe — CHANGES_REQUESTED** [review]

**Review by nemetabe — APPROVED** [review]
LGTM

**Pull request merged**